context/app/static/js/components/Detail/Citation/Citation.spec.js

```diff
@@ -1,14 +1,29 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import { render, screen } from 'test-utils/functions';
+import { composeStories } from '@storybook/testing-react';
+
 import { buildNLMCitation } from './Citation';
+import * as stories from './Citation.stories';
+
+const { Default } = composeStories(stories);
+
+const defaultCitation = 'Aanders A, Banders B, Canders C. Something Science-y [Internet]. HuBMAP Consortium; 2018.';
 
 test('builds NLM citation', () => {
-  const contributors = [
-    { last_name: 'Aanders', first_name: 'Aanne' },
-    { last_name: 'Banders', first_name: 'Banne' },
-    { last_name: 'Canders', first_name: 'Canne' },
-  ];
-  const title = 'Something Science-y';
-  const timestamp = 1520153805000;
-  expect(buildNLMCitation({ contributors, title, timestamp })).toEqual(
-    'Aanders A, Banders B, Canders C. Something Science-y [Internet]. HuBMAP Consortium; 2018.',
+  const { contributors, citationTitle, create_timestamp } = Default.args;
+  expect(buildNLMCitation({ contributors, citationTitle, create_timestamp })).toEqual(defaultCitation);
```

Comment:
So now the defaults are configured in the storybook? Mixed feelings: if the assertion does fail, the default inputs are now in a different file, and it's that much harder to understand what went wrong.

Reply:
I think we should share the props somehow rather than defining them twice whether it's in the `spec` files, the `stories` files or something separate.

Reply:
From our conversation: Merging for now. We ultimately don't want to add too much machinery/composition to our stories and tests. We can render the actual story in tests, but should redefine props in each file when we can't just render a story. We're going to review the Storybook docs and get a better sense of best practices.